string.md

```diff
@@ -28,16 +28,16 @@ When declaring a `STRING`, format it as `'a1b2c3'`.
 ## Examples
 
 ~~~
-CREATE TABLE strings (a STRING PRIMARY KEY, b CHAR, c TEXT);
+CREATE TABLE strings (a STRING PRIMARY KEY, b CHAR(6), c TEXT);
 
 SHOW COLUMNS FROM strings;
-+-------+--------+-------+---------+
-| Field |  Type  | Null  | Default |
-+-------+--------+-------+---------+
-| a     | STRING | false | NULL    |
-| b     | STRING | true  | NULL    |
-| c     | STRING | true  | NULL    |
-+-------+--------+-------+---------+
++-------+-----------+-------+---------+
+| Field |  Type     | Null  | Default |
++-------+-----------+-------+---------+
+| a     | STRING    | false | NULL    |
+| b     | STRING(6) | true  | NULL    |
```

Comment:
Heh, we don't actually support `STRING(N)`, only `{CHARACTER,CHAR,VARCHAR}(n)`. This is a bug. Mind filing an issue.